tests/_fixtures/fixturesPages.ts

```diff
@@ -0,0 +1,82 @@
+import { test as base } from '@playwright/test';
+import { SignUpPage } from '../../src/ui/pages/auth/SignUpPage';
+import { LogInPage } from '../../src/ui/pages/auth/LogInPage';
+import { LookupPage } from '../../src/ui/pages/auth/LookupPage';
+import { HomePage } from '../../src/ui/pages/HomePage';
+import { OverviewPage } from '../../src/ui/pages/account/OverviewPage';
+import { AccountDetailsPage } from '../../src/ui/pages/account/AccountDetailsPage';
+import { OpenNewAccountPage } from '../../src/ui/pages/account/OpenNewAccountPage';
+import { RequestLoanPage } from '../../src/ui/pages/account/RequestLoanPage';
+import { TransferFundsPage } from '../../src/ui/pages/account/TransferFundsPage';
+import { BillPayPage } from '../../src/ui/pages/account/BillPayPage';
+import { FindTransactionsPage } from '../../src/ui/pages/account/FindTransactionsPage';
+
+export const test = base.extend<{
+  signUpPage;
+  logInPage;
+  lookupPage;
+  homePage;
+  overviewPage;
+  accountDetailsPage;
+  openNewAccountPage;
+  requestLoanPage;
+  transferFundsPage;
+  billPayPage;
+  findTransactionsPage;
+}>({
```

Comment:
```suggestion
interface TestOptions {
  signUpPage;
  logInPage;
  lookupPage;
  homePage;
  overviewPage;
  accountDetailsPage;
  openNewAccountPage;
  requestLoanPage;
  transferFundsPage;
  billPayPage;
  findTransactionsPage;
}

export const test = base.extend<TestOptions>({
```

may be easier to read